spec/integration/profiles/ldap.rb

```diff
@@ -43,7 +43,17 @@
 ldap = Net::LDAP.new(params)
 
 ##
-# Add a test user.
+# Add test users and admins group.
+
+# Prints the last operation result, and if it's not ok (neither 0 nor 68 (entry
+# already exists)), then it will raise an exception.
+def handle_ldap_result!(ldap, params)
+  puts "#{ldap.get_operation_result.message} (code #{ldap.get_operation_result.code})."
```

Comment:
On purpose?

Reply:
Yes. It's the old output, but now applied to all LDAP actions (i.e. if it fails on an LDAP insert, I want to know why :wink:)